axon-jms/src/main/java/com/sevensenders/axon/jms/message/AxonMessageConverter.java

```diff
@@ -76,4 +76,15 @@ public Optional<EventMessage<?>> readMessage(JMSMessage message) {
             return Optional.of(new GenericEventMessage<>(eventMessage, () -> DateTimeUtils.parseInstant(timestamp)));
         }
     }
+    
+    public void push_message(Message message){
+      if(message != null)
+      {
```

Comment:
Can you please add space after if and curly bracket in same line with if statement. please use 4 space as formatting